experanto/experiment.py

```diff
@@ -27,15 +40,23 @@ def _load_devices(self) -> None:
 
         for d in device_folders:
             log.info(f"Parsing {d.name} data... ")
-            dev = Interpolator.create(d)
```

Comment:
With regard to my config dict suggestion above,  it will simplify to:


    def _load_devices(self) -> None:
        # Populate devices by going through subfolders
        # Assumption: blocks are sorted by start time
        device_folders = [d for d in self.root_folder.iterdir() if d.is_dir()]

        for d in device_folders:
            if d not in self.interp_config:
                continue
            log.info(f"Parsing {d.name} data... ")
            dev = Interpolator.create(d, self.interp_config)
            self.devices[d.name] = dev
            self.start_time = dev.start_time
            self.end_time = dev.end_time
            log.info("Parsing finished")